### ISSUE TITLE
ToolBar

### ISSUE THREAD

**Issue by MGD1** [memspace/zefyr#233]
I would like to dismiss the toolbar from the phone instead of the always using the ToolBar. When i try to dismiss it from the phone, it drops to the bottom. is this expected behavior or something that could be fixed?

Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, v1.12.13+hotfix.5, on Microsoft Windows [Version 10.0.18362.535], locale en-US)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[√] Android Studio (version 3.5)
[√] VS Code (version 1.41.1)
[√] Connected device (1 available)

• No issues found!
![Screenshot_20200116-153754](https://user-images.githubusercontent.com/23202122/72565515-a349ae00-3877-11ea-93e2-629c09df7712.jpg)

**Comment by cgestes** [comment]
are you speaking of the last button on the toolbar?
It is intended to dismiss the keyboard

**Comment by MGD1** [comment]
Yes correct but I would like to dismiss it on phone level. When I hit the
back arrow on the phone to get out of the keyboard. The toolbar drops to
the bottom of the screen and does not dismiss

**Comment by imikay** [comment]
Maybe this can help you? #265

**Comment by sumitbhanushali** [comment]
facing same issue, any solution to this?

**Comment by cgestes** [comment]
drop the focus from the editor maybe?

**Comment by jclink** [comment]
I ran into this problem, and for me, it appeared that the keyboard hiding event was absorbing the back button and I was unable to hide the Zefyr toolbar.  I managed to close the toolbar by using didChangeMetrics() and a bool for keyboard open;  
`bool _keyboardOpen = false;

  @override
  void initState() {
    super.initState();
    WidgetsBinding.instance.addObserver(this);
  }

  @override
  void dispose() {
    WidgetsBinding.instance.removeObserver(this);
    super.dispose();
  }


  @override
  void didChangeMetrics() {
    super.didChangeMetrics();
    final value = WidgetsBinding.instance.window.viewInsets.bottom;
    if(value > 0){
      _keyboardOpen = true;
    }
    if (_keyboardOpen && value == 0) {
      FocusScope.of(context).unfocus();
      _keyboardOpen = false;
    }
  }`
